packages/modules/create-imodel/src/components/base-imodel/BaseIModel.tsx

```diff
@@ -226,7 +225,7 @@ export function BaseIModelPage(props: BaseIModelProps) {
   const PointInput = (label: string, coordinate: keyof iModelExtent) => {
     return (
       <div className="iui-input-container">
-        <Label>{label}</Label>
+        <div className="iui-label">{label}</div>
```

Comment:
I think we should keep the Label since it is available in V2

Reply:
It breaks style as its inline, so decided to revert as it was..